tools/clang/lib/SPIRV/TypeTranslator.cpp

```diff
@@ -14,6 +14,13 @@ namespace clang {
 namespace spirv {
 
 uint32_t TypeTranslator::translateType(QualType type) {
+  {
```

Comment:
I'm not a fan of the extra braces here. I think the function would be easier to read without them.